### PR TITLE
Add our projects section to the landing page

### LINKED ISSUE #718: Create "Our projects" section
**Is your feature request related to a problem? Please describe.**
- We need to create a whole new section showing our projects.

**Describe the solution you'd like**
- Use a card displaying the logo, small description and the stats

**Additional context**
### Use `landing-page` branch

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #718 

## Goals

- Add our projects section to the landing page

## Approach

- Create a card
- Add the logo, description and the stats

### Screenshots

  
Screenshot from 2020-06-14 10-04-14

### Preview Link

https://pr-719-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Piumal1999** [comment]
@anjula-sack could you please link the relevent pages to the cards?

**Comment by anjula-sack** [comment]
Sure @Piumal1999

File: src/main/webapp/assets/css/global-custom.css [hunk 1]
```diff
@@ -540,3 +540,25 @@ body{
...
 }
+
+/*our projects display section*/
+.project-img {
+    padding: 15px;
+    width: 200px;
+    height: auto;
+}
+
+.project-card{
```
Comment (Piumal1999) [inline src/main/webapp/assets/css/global-custom.css]: I think this is not formatted. Can you please check that?

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

File: src/main/webapp/assets/css/global-custom.css (hunk 1 above)
Comment (anjula-sack) [reply src/main/webapp/assets/css/global-custom.css]: fixed

**Review by Piumal1999 — APPROVED** [review]
Well done! @anjula-sack

**Review by jayasanka-sack — APPROVED** [review]
Thanks @anjula-sack !

**Pull request merged**